src/js/intl/strings.js

```diff
@@ -359,7 +358,7 @@ exports.strings = {
     'es_AR': 'El repositorio remoto divergió de tu repositorio local, por lo que subir tus cambios no es un simple fast forward (y por eso se rechazó tu push). Por favor, hacé pull de los nuevos cambios en el repositorio remoto, incorporalos a esta rama y probá de nuevo. Podés hacerlo con git pull o git pull --rebase',
     'es_MX': 'El repositorio remoto divergió de tu repositorio local, por lo que subir tus cambios no es un simple fast forward (y por eso se rechazó tu push). Por favor, hacé pull de los nuevos cambios en el repositorio remoto, incorporalos a esta rama y prueba de nuevo. Puedes hacerlo con `git pull` o `git pull --rebase`',
     'es_ES': 'El repositorio remoto se ha desviado del repositorio local. Subir cambios no es un avance en cámara rápida (por eso fue rechazado tu push). Por favor haz pull para descargar los cambios en el repositorio remoto para que los incorpores en la rama actual. Los cambios se pueden hacer pull con el comando "git pull" o "git pull --rebase"',
-    'pt_BR': 'O repositório remoto divergiu do repositório local, então enviar suas mudanças não é um simples fast forward (e por isso seu push foi rejeitado). Por favor, faça pull das novas mudanças do repositório remoto, incorpore-os a este ramo, e tente novamente. Você pode fazê-lo com git pull ou git pull --rebase',
+    'pt_BR': 'O repositório remoto divergiu do repositório local, então enviar suas mudanças não são um simples fast forward (e por isso seu push foi rejeitado). Por favor, faça pull das novas mudanças do repositório remoto, incorpore-os a este ramo, e tente novamente. Você pode fazê-lo com git pull ou git pull --rebase',
```

Comment:
I disagree with this change. The verb "é" was referring to "enviar", not to "mudanças", so the singular form was the grammatically correct choice.

Reply:
I'm not a professional but this sounded wrong, so I used a tool https://languagetool.org/pt-BR to check the translation and it was marked as a gramatical error: "Possível erro de concordância verbal."

Reply:
Yes, it is a possible error if you're uncertain about what is the subject — the tool is correct in being cautious. But as a human who knows the language, I can say for sure that the subject is "enviar", or in other words, "o envio" — which is singular. So again, it should remain singular.

Reply:
Ok I agree.